src/app.js

```diff
@@ -1 +1,203 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from 'axios';
+
+console.log('Hallo daar!');
+const ulElement = document.getElementById('countries');
+
+async function getAll() {
+    try {
+        const response = await axios.get('https://restcountries.com/v2/all');
+        // console.log(response.data);
+        return response.data;
+    } catch (e) {
+        console.error(e);
+    }
+}
+
+function doListOfCountries(){
+    getAll()
+        .then((result) => {
+            sortLandsByPopulation(result);
+            result.map((land) => {
```

Comment:
Je kunt hier nog spreading toepassen, zodat je niet overal `lan.` vóór hoeft te zetten. Voorbeeldje:
```suggestion
            result.map(({ flags, name, region, population }) => {
```